engine/events/halloffame.asm

```diff
@@ -376,7 +376,7 @@ _HallOfFamePC: ; 86650
 	pop hl
 	call DisplayHOFMon
 	ld a, [wHallOfFameTempWinCount]
-	cp 200 + 1
+	cp HOF_MASTER_COUNT
```

Comment:
I think this is backwards. The compare should be `HOF_MASTER_COUNT + 1` and the counter should stop at `HOF_MASTER_COUNT`.

Reply:
I was uncertain about this too, but the video shows that the Japanese version displayed "200-Time Famer" and then "HOF Master" for 201 wins.

Reply:
Before I patched the Japanese ROM, it'd display the equivalent to "200-Time Famer" every time for the 201th entry and later, not only the 200th. Changing `cp 200 + 1` there to `cp 200` fixed it for me.

Reply:
Before I patched the Japanese ROM, it'd display the equivalent to "200-Time Famer" every time for the 201st entry and later, not only the 200th. Changing `cp 200 + 1` there to `cp 200` fixed it for me: if the counter is at 199, entering the Hall of Fame 200 times will update the counter to 200 and use the normal message and, once at 200, any new entry (201+) displays the special message.

Reply:
Oh, I didn't realize the Japanese one had to be patched too. In that case, yeah, special-casing 200 makes more sense than 201. Thanks.